docs/custom_node_development.md

```diff
@@ -81,6 +81,14 @@ Custom nodes can generate the results which have dynamic size depending on the i
 In such case, function `getInputsInfo` should return value `0` on the dimension with dynamic size. It could be input with
 variable resolution or batch size. 
 
+**String inputs support**
```

Comment:
This does not look good when md is parsed:
https://github.com/openvinotoolkit/model_server/blob/af0fe36fa2ba5e488a4ef22d4fc6114c46ac788f/docs/custom_node_development.md#getinputsinfo-function

Also I dont think this is the best place to put such information. Yes, this is useful information in terms of `getInputsInfo` but also `getOutputsInfo` - node can return 2d u8 and have the OVMS convert that to strings when end node name is suffixed with `_string` - this is entirely missing.

It is also important to understand of `execute` function to understand how to read input/prepare output data.

My suggestion is to create separate paragraph like we do with `Using OpenCV` explaining all in details.